scrapeArtists/server.js

```diff
@@ -0,0 +1,56 @@
+var request = require("request");

```

Comment:
Is this script still needed? It is not getting called from /musicbrainz/musicbrainzcli/server.js